pymdownx/tasklist.py

```diff
@@ -30,17 +30,18 @@
 
 RE_CHECKBOX = re.compile(r"^(?P<checkbox> *\[(?P<state>(?:x|X| ){1})\] +)(?P<line>.*)")
 
-
-def get_checkbox(state, custom_checkbox=False):
+def get_checkbox(state, custom_checkbox=False, clickable_checkbox=False):
     """Get checkbox tag."""
 
     if custom_checkbox:
         return (
             '<label class="task-list-control">' +
-            '<input type="checkbox" disabled%s/>' % (' checked' if state.lower() == 'x' else '') +
+            '<input type="checkbox"%s%s/>' % (' disabled' if clickable_checkbox == False else '',
```

Comment:
Tests are failing because you added a trailing self closing `/` in the input tag.  This is probably better so that when people insert this into xhtml (but who is doing this these days).  So I'm fine with the change, but you need to regenerate changed, failing tasklist tests.